extension/css/inspector/partials/inspector/_inspector-panel.scss

```diff
@@ -1,7 +1,3 @@
-html, body {
-    height: 100%;
-}
-
```

Comment:
This I removed just because we have these style elsewhere. Happened to find it while I was working on this.